examples/vision/aisio_usage_example.ipynb

```diff
@@ -0,0 +1,321 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# loading data using AISFileLoader\n",
+    "dp_files = AISFileLoader(url=\"http://localhost:51080\", source_datapipe=dp_urls)\n",
+    "\n",
+    "# check the first obj\n",
+    "url, img = next(iter(dp_files))\n",
+    "\n",
+    "print(f\"image url: {url}\")\n",
+    "Image(data=img.read())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "dd521f6a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def collate_sample(data):\n",
+    "    path, image = data\n",
+    "    dir = os.path.split(os.path.dirname(path))[1]\n",
+    "    label_str, cls = dir.split(\".\")\n",
+    "    return {\"path\": path, \"image\": image, \"label\": int(label_str), \"cls\": cls}"
```

Comment:
IIRC, AIS does support offload transformation to the server side. Let's say we have such collate function for the ETL pipeline, could AIS server runs this collate function on the server side? If so and we delegate this collate function to the server side, would we have performance gain?